File: java/src/main/java/com/iteso/nintendo/Surf.java
package com.iteso.nintendo;
/**Ataque Surf.*/
public class Surf implements Attack {

    /**Nombre del ataque.*/
    private final String name = "Ataque Surf";
    /**Daño del ataque.*/
    private final int damage = 10;

    /**@return Nombre del ataque.*/
    @Override
    public String getAttackName() {
        return this.name;
    }

    /**@return Daño del ataque.*/
    @Override
    public int getAttackDamage() {
        return this.damage;
    }
}

File: csharp/Clases/Surf.cs
namespace unittestpractice.Clases
{
    public class Surf : IAttack
    {

        /**Nombre del ataque.*/
        private const string Name = "Ataque Surf";
        /**Daño del ataque.*/
        private const int Damage = 10;

        public string GetAttackName()
        {
            return Name;
        }

        public int GetAttackDamage()
        {
            return Damage;
        }
    }
}